pkg/stagemanager/stage_manager.go

```diff
@@ -667,3 +652,62 @@ func (stm *StageManager) StopMosnProcess() (err error) {
 		return
 	}
 }
+
+// ReloadMosnProcess stops Mosn process via command line,
+// and it not support reload on windows.
+func (stm *StageManager) ReloadMosnProcess() (err error) {
+	// init
+	stm.runParamsParsedStage()
+	stm.runReconfigurationParamsInit()
+
+	mosnConfig := stm.data.config
+
+	log.StartLogger.Infof("[mosn reload] start reload mosn configurations")
+	// finds process and sends SIGINT to mosn process, makes it force exit.
+	proc, p, err := getMosnProcess(mosnConfig)
```

Comment:
`p` is a bad name, better to use `pid` instead?

Reply:
pid is a package imported and I already changed the var name as `procid`